Algorithms/check_permutation.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# CtCI 6th Edition Problem 1.2
+# Given two strings, write a method to decide if one is a permutation of the
+# other.
+
+from timeit import timeit
+import collections
+
+
+def is_permutation(string1, string2):
+    return (len(string1) == len(string2)
+            and sorted(string1) == sorted(string2))
```

Comment:
1. Python has really strict style guidelines, and I'm not familiar with them, so good on you! I only suggested it because of the [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains) on multi line chains (Option B).
2. Okay, it looked to me like it might not have been over the 80 character limit. I didn't view it in my editor though, so I won't make any more comments on that.